src/render/views/CreateActorTemplateView.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import {
+    Box,
+    Button,
+    createStyles,
+    FormControlLabel,
+    makeStyles,
+    Typography
+} from "@material-ui/core";
+import { Cancel as CancelIcon, Save as SaveIcon } from "@material-ui/icons";
+import { Checkbox, TextField } from "@/render/components";
+import { ActorType } from "@/render/database/schema";
+import { getActorTemplateManager } from "@/render/state/ActorTemplateManager";
+import { getViewManager, View } from "@/render/state/ViewManager";
+import { css, useObservable } from "@/util";
+
+const useStyles = makeStyles(theme =>
+    createStyles({
+        buttonRow: {
+            marginTop: theme.spacing(1)
+        },
+        firstButton: {
+            marginLeft: "auto"
+        },
+        notLastButton: {
+            marginRight: theme.spacing(1)
+        }
+    })
+);
+
+export const CreateActorTemplateView: React.FC<{}> = () => {
+    const styles = useStyles();
+    const [name, setName] = useObservable("");
+    const [initiativeModifier, setInitiativeModifier] = useObservable("");
+    const [isUniqueName, setIsUniqueName] = useObservable(false);
+    const [actorType, setActorType] = useObservable(ActorType.NPC);
```

Comment:
I spent several hours trying to find a good way to make this one declaration, but alas I am bad and it did not work.

Reply:
all good my man